sdk/communication/communication-rooms/src/roomsClient.ts

```diff
@@ -234,26 +219,10 @@ export class RoomsClient {
     pageSettings: ListPageSettings,
     options: ListParticipantsOptions = {},
   ): AsyncIterableIterator<RoomParticipant[]> {
-    if (!pageSettings.continuationToken) {
-      const currentSetResponse = await this.client.participants.list(roomId, options);
-      pageSettings.continuationToken = currentSetResponse.nextLink;
-      if (currentSetResponse.value) {
-        yield currentSetResponse.value.map(mapToRoomParticipantSDKModel, this);
-      }
-    }
-
-    while (pageSettings.continuationToken) {
-      const currentSetResponse = await this.client.participants.listNext(
-        roomId,
-        pageSettings.continuationToken,
-        options,
-      );
-      pageSettings.continuationToken = currentSetResponse.nextLink;
-      if (currentSetResponse.value) {
-        yield currentSetResponse.value.map(mapToRoomParticipantSDKModel, this);
-      } else {
-        break;
-      }
+    const currentSetResponse = await this.client.participants.list(roomId, options);
```

Comment:
This section was changed , as the autogen changed the signature of list method from Promise to PagedAsyncIterableIterator. We previously had to iterate with the next link and now with the PagedAsyncIterableIterator we can use the for await